.dependabot/config.yml

```diff
@@ -0,0 +1,5 @@
+version: 1
+update_configs:
+- package_manager: go:modules
+  directory: /
+  update_schedule: daily
```

Comment:
How about `update_schedule: weekly` ? `daily` might be too frequently.

Reply:
ok.